Lombiq.OrchardCoreApiClient/ConfigurableCertificateValidatingHttpClientHandler.cs

```diff
@@ -63,7 +63,7 @@ protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage
             };
 #pragma warning restore CA5400
 
-            var tokenResponse = await RestClient
+            var tokenResponse = await RestService
```

Comment:
I start OSOCE, import `Lombiq.OrchardCoreApiClient.Tester.OpenId.recipe` and get this exception:

![image](https://github.com/Lombiq/Orchard-Core-API-Client/assets/1976647/48769527-919f-43af-82b5-f6095faa5a16)

Reply:
Yes, with Refit all paths have to start with slash even if the base URL isn't just a domain. I've fixed this on the other two APIs but looks like I missed _IOrchardCoreAuthorizationApi.cs_. Now double checked, all paths are updated.

Reply:
Now I get this:

![image](https://github.com/Lombiq/Orchard-Core-API-Client/assets/1976647/b19fd103-e2d8-4271-ab6f-7f0c17f1caad)

Reply:
This is fixed. It was because it uses System.Text.Json by default. So I actually had to create a Refit library in HL after all, to avoid having to configure this everywhere.

But past that, is this tester actually supposed to work with OSOCE? Because it still fails [at the end](https://github.com/Lombiq/Orchard-Core-API-Client/blob/dev/Lombiq.OrchardCoreApiClient.Tester/Program.cs#L65) because OrchardCoreApi.EditAsync throws a 404 error. When I check the OC source code, this [Edit API endpoint](https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/ApiController.cs#L146) exists in dev [and in 1.6.0](https://github.com/OrchardCMS/OrchardCore/blob/v1.6.0/src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/ApiController.cs#L146), but we are still using OC [1.5.0 where it's not yet implemented](https://github.com/OrchardCMS/OrchardCore/blob/v1.5.0/src/OrchardCore.Modules/OrchardCore.Tenants/Controllers/ApiController.cs).

Reply:
I created a basic OC 1.6.0 CMS project from template, and imported the recipe there. It works there.

Reply:
Yes, the edit method is only for DotNest Tenants Core yet, since it uses a post-1.5 nightly, but otherwise the tester should work.